metals/src/main/scala/scala/meta/internal/metals/BloopServers.scala

```diff
@@ -121,6 +127,182 @@ final class BloopServers(
     }
   }
 
+  private def writeJVMPropertiesToBloopGlobalJsonFile(
+      bloopGlobalJsonFile: File,
```

Comment:
```suggestion
      bloopGlobalJsonFile: AbsolutePath,
```

this would allow to use `.writeText` extension method without the additional BufferedWriter etc.

Reply:
Good hint! Thanks, I would use it...